src/JuMP.jl

```diff
@@ -378,21 +379,25 @@ See also: [`backend`](@ref).
 This function is unsafe because the innermost optimizer may not represent the
 same problem as `model`.
 
-In particular, if you modify the JuMP model, the reference you have to the
-backend may be invalid. Using an invalid reference may result in incorrect
-solutions, errors, or Julia crashing. Always get a new reference by calling
-`unsafe_backend` before calling any low-level functionality.
+If you modify the JuMP model, the reference you have to the backend may be
+invalid. Using an invalid reference may result in incorrect solutions, errors,
+or Julia crashing. Always get a new reference by calling `unsafe_backend` before
+calling any low-level functionality.
 
-Moreover, if you pass `attach_caching_optimizers = false`, you should assume
-that any `CachingOptimizer`s may be in the `EMPTY_OPTIMIZER` state.
+Moreover, if you modify the unsafe backend, e.g., by adding a new constraint,
+the changes may be silently discarded by JuMP when the model is modified or
+solved. Don't to this unless you understand when and how JuMP attaches and
```

Comment:
to -> do

Reply:
Huh. Something is weird here. Looks like you couldn't see the latest changes. I already fixed this.